pkg/vault/contracts/Vault.sol

```diff
@@ -1183,29 +1189,26 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
 
             // ERC4626 output should not be trusted, so it's a good practice to measure the amount of
```

Comment:
Here and below, this comment is a bit stale, as it refers to the previous code that returned the deltas from `_updateReservesAfterWrapping`. Maybe reword and move these comments to the `_settleWrap` and `_settleUnwrap` functions?

Reply:
_settleWrap and _settleUnwrap already explain this, so I just removed it.